src/main/java/su/litvak/chromecast/api/v2/Channel.java

```diff
@@ -76,7 +76,7 @@
     /**
      * Counter for producing request numbers
      */
-    private AtomicLong requestCounter = new AtomicLong(0);
+    private AtomicLong requestCounter = new AtomicLong(1);
```

Comment:
Chromecast uses 0, for all Spontaneous messages.
https://developers.google.com/cast/docs/reference/messages